### PR TITLE
Refractor radius attr

### PR DESCRIPTION
**Pull request by Bairdo**
Looks like there was an error on #54 this should fix it (also builds on top of it.)

### CODE REVIEW

**Comment by samrussell** [comment]
What's the logic behind this refactor? The idea of standardising the attributes and putting them into one place is a good one, but doing string lookups feels like a backwards step - we should be working with meaningful objects (consts, enums, classes etc) rather than passing in strings for a hash lookup.

**Comment by Bairdo** [comment]
It seemed like a good idea to support more than the bare minimum as used during our testing.

And the Attribute class interface was convoluted and hard to follow.

If a user can just swap out their traditional switch and put in a Faucet switch, without having to make any changes to their RADIUS server then that seemed like a good idea.

And i guess i thought the string version looked nicer than 200 classes. If you could point me in a better direction that would be awesome.
